rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/rest/RestHandler.java

```diff
@@ -49,24 +49,41 @@
     public RestHandler(ConnectController connectController) {
         this.connectController = connectController;
         Javalin app = Javalin.start(connectController.getConnectConfig().getHttpPort());
-        app.get("/connectors/stopAll", this::handleStopAllConnector);
-        app.get("/connectors/pauseAll", this::handlePauseAllConnector);
-        app.get("/connectors/resumeAll", this::handleResumeAllConnector);
-        app.get("/connectors/enableAll", this::handleEnableAllConnector);
-        app.get("/connectors/disableAll", this::handleDisableAllConnector);
-        app.get("/connectors/:connectorName", this::handleCreateConnector);
-        app.get("/connectors/:connectorName/config", this::handleQueryConnectorConfig);
-        app.get("/connectors/:connectorName/status", this::handleQueryConnectorStatus);
-        app.get("/connectors/:connectorName/stop", this::handleStopConnector);
-        app.get("/connectors/:connectorName/pause", this::handlePauseConnector);
-        app.get("/connectors/:connectorName/resume", this::handleResumeConnector);
-        app.get("/connectors/:connectorName/enable", this::handleEnableConnector);
-        app.get("/connectors/:connectorName/disable", this::handleDisableConnector);
-        app.get("/getClusterInfo", this::getClusterInfo);
-        app.get("/getConfigInfo", this::getConfigInfo);
-        app.get("/getAllocatedConnectors", this::getAllocatedConnectors);
-        app.get("/getAllocatedTasks", this::getAllocatedTasks);
-        app.get("/plugin/reload", this::reloadPlugins);
+        if (this.connectController.getConnectConfig().getIsLeader() == 1){
+            app.get("/connectors/stopAll", this::handleStopAllConnector);
+            app.get("/connectors/pauseAll", this::handlePauseAllConnector);
+            app.get("/connectors/resumeAll", this::handleResumeAllConnector);
+            app.get("/connectors/enableAll", this::handleEnableAllConnector);
+            app.get("/connectors/disableAll", this::handleDisableAllConnector);
+            app.get("/connectors/:connectorName", this::handleCreateConnector);
+            app.get("/connectors/:connectorName/config", this::handleQueryConnectorConfig);
+            app.get("/connectors/:connectorName/status", this::handleQueryConnectorStatus);
+            app.get("/connectors/:connectorName/stop", this::handleStopConnector);
+            app.get("/connectors/:connectorName/pause", this::handlePauseConnector);
+            app.get("/connectors/:connectorName/resume", this::handleResumeConnector);
+            app.get("/connectors/:connectorName/enable", this::handleEnableConnector);
+            app.get("/connectors/:connectorName/disable", this::handleDisableConnector);
+            app.get("/getClusterInfo", this::getClusterInfo);
```

Comment:
I think slave nodes could serve read-only requests, it just cannot update config, what do you think ~

Reply:
Yes, it's a lazy way. I have updated it in my new commit.